hack/common.sh

```diff
@@ -34,24 +39,28 @@ OCS_CSV="$OUTDIR_TEMPLATES/ocs-operator.csv.yaml.in"
 DEFAULT_IMAGE_REGISTRY="quay.io"
 DEFAULT_REGISTRY_NAMESPACE="ocs-dev"
 DEFAULT_IMAGE_TAG="latest"
+DEFAULT_OPERATOR_IMAGE_NAME="ocs-operator"
+DEFAULT_CATALOG_IMAGE_NAME="ocs-registry"
+DEFAULT_MUST_GATHER_IMAGE_NAME="ocs-must-gather"
 
 IMAGE_REGISTRY="${IMAGE_REGISTRY:-${DEFAULT_IMAGE_REGISTRY}}"
 REGISTRY_NAMESPACE="${REGISTRY_NAMESPACE:-${DEFAULT_REGISTRY_NAMESPACE}}"
-IMAGE_NAME="ocs-registry"
-IMAGE_TAG="${IMAGE_TAG:-latest}"
-FULL_IMAGE_NAME="${IMAGE_REGISTRY}/${REGISTRY_NAMESPACE}/${IMAGE_NAME}:${IMAGE_TAG}"
+OPERATOR_IMAGE_NAME="${OPERATOR_IMAGE_NAME:-${DEFAULT_OPERATOR_IMAGE_NAME}}"
+CATALOG_IMAGE_NAME="${CATALOG_IMAGE_NAME:-${DEFAULT_CATALOG_IMAGE_NAME}}"
+MUST_GATHER_IMAGE_NAME="${MUST_GATHER_IMAGE_NAME:-${DEFAULT_MUST_GATHER_IMAGE_NAME}}"
+IMAGE_TAG="${IMAGE_TAG:-${DEFAULT_IMAGE_TAG}}"
```

Comment:
Not a big deal, but if we put the IMAGE_TAG var earlier on, we can group CATALOG_IMAGE_NAME and FULL_CATALOG_IMAGE_NAME and the corresponding pair for MUST_GATHER, which seems useful. Absolutely a matter of taste though.